packages/router/src/Exceptions/SendsResponse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tempest\Router\Exceptions;
+
+use Tempest\Http\Response;
+
+/**
+ * Marks this exception class as one that can be converted to a response.
+ *
+ * @phpstan-require-extends \Throwable
+ */
+interface SendsResponse
```

Comment:
Nitpick: it's not this class that _sends_ the response, this class can be converted _to_ a response. Maybe we need a slightly better name, although I'm not sure what that should be.

- `ConvertsToResponse`
- Laravel has `Responsable`, which is a really bad name
- Something else?